### PR TITLE
Add chibi@1.1.5 with git autoupdate.

### PR DESCRIPTION
**Pull request by Piicksarn**
@Amomo , would you mind to have a look at the pr?
The PR is referenced to issue  #5928.
Thank you!
The information of the pr:
The git repo : https://github.com/kylebarrow/chibi/tree/1.1.5

- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity : **Watch `25`, Star `403`, Fork `24`**
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
chibi-min.js or chibi.min.js?

File: ajax/libs/chibi/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "chibi",
+  "filename": "chibi.min.js",
```
Comment (Amomo) [inline ajax/libs/chibi/package.json]: They has `chibi-min.js`.

**Comment by Amomo** [comment]
@Piicksarn could you please update your PR~?

**Comment by Piicksarn** [comment]
@Amomo, would you mind to checkout the pr again?
Thank you~~

**Comment by Amomo** [comment]
@Piicksarn I think this PR is fine, could you rebase it?

**Comment by Piicksarn** [comment]
@Amomo, i have it rebased!
Would you mind to have a look at it again?~thank you~

**Comment by Amomo** [comment]
@Piicksarn don't forget to "fetch" again before you rebase your branch.
And, some of your PR haven't updated for a while, don't forget them~

@PeterDaveHello I think this PR is fine, could you take a look? Thanks~

**Pull request merged**